Libraries/LibWebView/Application.cpp

```diff
@@ -27,7 +27,8 @@ namespace WebView {
 Application* Application::s_the = nullptr;
 
 Application::Application()
-    : m_settings(Settings::create({}))
+    : SettingsObserver(AddToObservers::No) // Application::the() is not set yet.
```

Comment:
Oh yeah, that is unfortunate. This is fine for now, but we should come up with something more consistent another time.